examples/llama_recipes/main.py

```diff
@@ -0,0 +1,110 @@
+from fastapi import FastAPI
```

Comment:
Not your fault but am wondering if we should make make the sample submission a library of sorts, in all cases I think this is fine since competitors are likely to do the same

Reply:
I think I see what you mean, like
```
from neurips_llm_efficiency_challenge.api import (ProcessRequest,
    ProcessResponse,
    TokenizeRequest,
    TokenizeResponse,
    Token,
)
```
And some thin wrapper for the contextmanager
```
@app.post("/process")
```
But I am not sure if its worth the effort as competitors are likely to just grab one of the examples and fit their needs.